### PR TITLE
Update connections-default.xml

### PR DESCRIPTION
**Pull request by pvgenuchten**
this is a list of csw servers based on geonetwork, i discovered them a couple of months back, some of them might not be online anymore, i wanted to write a script to check that automatically. and promote them in the list based on % availability, from my blog post: http://genuchten.blogspot.nl/2013/06/amazing-list-of-geonetworks-around-globe.html

### CODE REVIEW

**Comment by tomkralidis** [comment]
Thanks @pvgenuchten.  I echo @rduivenvoorde.  FYI part of our CI includes testing every CSW that is in `plugin/MetaSearch/resources/connections-default.xml`.  If the CSW is problematic (against a simple GetCapabilities test, which if it cannot pass, then there are real problems :) ), the build fails.

Having said this, I also think that only stable, major CSW endpoints should be in the default list (although coming up with a process for this might be a bit much).  For example, I am putting together a big pycsw-based CSW list, but I don't think it should be in the default MetaSearch list.

Here's what I propose, for review and comment:
- default connections list provides a list of authoritative (preferably in national scope) CSW servers
- servers are listed by CSW server name (taken from `csw:Capabilities/ows:ServiceIdentification/ows:Title`) in alphabetical order
- servers can be proposed for inclusion to the default list via GitHub issues
- any server causing the build to break is removed
- all other servers can be listed either:
  - on a MetaSearch wiki page as an XML file that someone can copy/paste
  - in `plugin/MetaSearch/resources/connections-contrib.xml` in which the user can navigate to and add Load the whole list, or add by hand from the XML
  - given CSW implementations (pycsw, GeoNetwork, deegree2, ESRI Geoportal, etc.) can host their own CSW collection XML elsewhere and link them off MetaSearch wiki

In summary, I think having a "contributed" list of CSWs would be very valuable, but too volatile/error prone to keep in the codebase (and chasing broken builds), so I propose posting these to the MetaSearch wiki which will let us manage with ease.

Comments @pvgenuchten, @geopython/metasearch team?

**Comment by tomkralidis** [comment]
FYI closing.  I've put this on wiki (https://github.com/geopython/MetaSearch/wiki/MetaSearch-Default-Connections-List#wiki-guidance-for-adding-csws-to-the-default-list) as way forward for adding endpoints.
